apollo-router/src/plugins/external.rs

```diff
@@ -43,41 +43,51 @@ struct ExternalPlugin {
     sdl: Arc<String>,
 }
 
+/// The
 #[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize, JsonSchema)]
 struct BaseConf {
+    /// Send the headers
```

Comment:
In general, I don't think we've added cargo doc comments for internal code. Is this because the orbiter code is extracting them somehow?

Reply:
It does, but in addition it forms the docs in the generated json schema that provides feedback to the user via autocomplete.
From now on we should provide high quality cargo docs for all config.